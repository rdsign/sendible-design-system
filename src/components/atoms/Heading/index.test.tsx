import { create } from 'react-test-renderer';
import { Heading } from '.';
import { Template } from '../../../../spec/template';

describe('Icon', () => {
    const component = (
        <Template>
            <Heading name="h1" size="s80">
                Sendible<i> Rocks!</i>
            </Heading>
        </Template>
    );

    test('should match snapshot', () => {
        const snapshot = create(component);
        expect(snapshot).toMatchSnapshot();
    });
});
