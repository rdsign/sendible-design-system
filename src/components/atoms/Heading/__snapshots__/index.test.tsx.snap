// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should match snapshot 1`] = `
.c0 {
  color: #201E24;
  font-size: 5rem;
  font-weight: 500;
  line-height: 1em;
}

.c0 i {
  color: #FF6153E6;
  font-weight: 400;
}

<h1
  className="c0"
  name="h1"
  size="s80"
>
  Sendible
  <i>
     Rocks!
  </i>
</h1>
`;
